### DISCUSSION TITLE
dual-licensing default files

### DISCUSSION THREAD

**Discussion by dezren39** [srid/emanote#545] [Ideas]
hello are you amenable to dual-licensing specifically
- `emanote/default/index.yaml`
- `emanote/default/templates/*`

in order to better be able to support users forking `emanote-template` and then pulling in templates to customize their usage.

my concern is that while `emanote-template` is CC0, a user may update their `after-note.tpl` from `emanote-template`, all good, then they want to modify `sidebar.tpl` or `sidebar-tree.tpl` etc. and then they need to choose if they are going to relicense or how they share out their code changes to comply with agpl. a naive user may "just copy" the file into their `emanote-template` while following suggestions in docs here/elsewhere without realizing the complexities involved in licensing, violating the agpl license by making the file appear cc0. if a user looks at `emanote/default/index.yaml` in order to understand how best to modify their own then that might need to be relicensed too?

possible options i see..
- do nothing, except maybe document this for users
- dual-license above files with cc0 or mit or something. mpl-2.0 whatever
- copy above files into `emanote-template` repo
  - this has some unfortunate complexities,
    - the 2 instances of the file could get out of sync
    - would maybe still need some documentation so users know their options

i might just move my default files back into my emanote fork, edit the defaults directly and then that will thread the needle for me for now.

**Comment by srid** [comment]
> dual-license above files with cc0 or mit or something. mpl-2.0 whatever

This works for me

**Comment by dezren39** [comment]
here's a first attempt at clarifying licensing:

- https://github.com/srid/emanote/pull/549

happy to change however you see fit or just close and let you get to it however you prefer when you've got time. :-)
